CorsixTH/Lua/humanoid_actions/use_object.lua

```diff
@@ -386,12 +386,13 @@ action_use_object_tick = permanent"action_use_object_tick"( function(humanoid)
 
     -- Check if the room is about to be destroyed
     local room_destroyed = false
-    if object.strength then
+    if object:isMachine() then
       if humanoid.humanoid_class ~= "Handyman"  then
         room_destroyed = object:machineUsed(humanoid:getRoom())
       end
     elseif object:getDynamicInfo() and not object.master then
       -- Don't update if it is a slave object.
```

Comment:
`not object.master` is tricky here. It looks wrong at first sight, but it's actually correct. I think that could use a comment.